recipe/bld.bat

```diff
@@ -9,8 +9,6 @@ cd build
 cmake ^
   -G "Ninja" ^
   %CMAKE_ARGS%                              ^
-  -DUSE_CLING=OFF                           ^
-  -DUSE_REPL=ON                             ^
```

Comment:
We have REPL on by default now so these would be ignored.

Reply:
Probably not if we decided to use older llvm but let’s move forward.